boardswarm-cli/src/main.rs

```diff
@@ -528,14 +534,19 @@ async fn run_configure(
         .as_ref()
         .map_or(false, |s| matches!(s.auth, config::Auth::Token(_)))
     {
+        info!("Using current server: {:?}", current_server);
         None
     } else {
         // OIDC
+        info!("Using OIDC authentication");
         let mut boardswarm = BoardswarmBuilder::new(uri.clone()).connect().await?;
         let info = boardswarm.login_info().await?;
 
         // TODO allow user select the !first one
-        let login = info.first().unwrap();
+        let login = info.first();
```

Comment:
perhaps context may be better work:
```suggestion
        let login = info.first().context("No OIDC authentication to choose from")?
```

Reply:
thx, I will apply this suggestion

Reply:
However, due to "cargo fmt" it won't use less lines :P

Reply:
i just prefer `context` as its just a bit cleaner, i dont care about line count 

(note: don't take rust advice from me, i am simply a noob :p)